metrics/system.go

```diff
@@ -32,7 +32,11 @@ func CPUTime() (float64, error) {
 // goroutine.
 func CaptureCPUTime(interval time.Duration) {
 	cpuUtilizationGauge := metrics.NewGaugeFloat64()
-	metrics.Register("runtime.CpuStats.Utilization", cpuUtilizationGauge)
+	err := metrics.Register("runtime.CpuStats.Utilization", cpuUtilizationGauge)
```

Comment:
How is this change related to travis?

Reply:
That is what was making the build fail when linter was ran.